spikes/lk_boost_ut/tests/test_tag.cpp

```diff
@@ -0,0 +1,29 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "./tests/config.h"
+#include "boost/ut.hpp"
+
+using namespace boost::ut;
+
+// ? Example taken from:
+// ? https://github.com/boost-ext/ut/blob/master/example/tag.cpp
+
+suite suite_tag = [] {
+	ut::cfg<override> = {
+		.filter = "tag", .tag = {"execute"}, .colors = {.none = "", .pass = "", .fail = ""}, .dry_run = false};
+
+	tag("execute") / skip / "tag - good + skip - skipped"_test = [] {
+		expect(42_i == 43) << "should not fire!";
+		expect(false) << "should fail!";
+	};
+
+	tag("execute") / "tag - good - executed"_test = [] { expect(42_i == 42); };
+
+	tag("not executed") / "tag - wrong - skipped"_test = [] { expect(43_i == 42); };
+
+	tag("execute") / tag("not executed") / "tag - good + wrong - skipped"_test = [] { expect(42_i == 42); };
+
+	tag("execute") / tag("another tag") / "tag - good + good - skipped"_test = [] { expect(42_i == 42); };
+};
```

Comment:
Something goes wrong with this suite, they are not all displayed and I don't understand the last expected test of the suite

Reply:
Removing as not useful in our use case